# orbslam2_ros/Data is linked to sample data directory
vocabulary_file_path: "/root/GIT/ROSWS/sim_ws/src/orbslam2_ros/Data/Vocabulary/ORBvoc.txt"
#setting_file_path: "/root/GIT/ROSWS/sim_ws/src/orbslam2_ros/Data/Example/RGB-D/TUM1.yaml"
# RGB-D/TUM1.yaml has invalid value of DepthMapFactor when it is used with rosbag?
setting_file_path: "/root/GIT/ROSWS/sim_ws/src/orbslam2_ros/params/rgbd_TUM1.yaml"
frame_id: "openni_rgb_optical_frame"
child_frame_id: "map"
interface_type: "rgbd"
visualization: true
